server_admin/topics/vault.adoc

```diff
@@ -0,0 +1,57 @@
+[[_vault-administration]]
+
+== Using Vault to Obtain Secrets
+
+Several fields in the administration support obtaining the value of a secret from an external vault.
+
+To obtain a secret from a vault instead of entering it directly, enter
+the following specially crafted string into the appropriate field:
+`**${vault.**_vault.entry.name_**}**` where you replace the `_vault.entry.name_`
```

Comment:
maybe "vault.entry-name" would be better? Using dots in vault.entry.name makes it a bit confusing as you may think it's a hierarchy of sorts

Reply:
+1